### PR TITLE
make sure bravery settings are consistent everywhere

### LINKED ISSUE #2174: bravery UX doesn't show number of scripts blocked when noscript is globally on
due to `siteSettings.activeSettings()` not accounting for global appState bravery settings.

### PR DESCRIPTION
**Pull request by bridiver**
- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [x] Ran `git rebase -i` to squash commits if needed.

fixes #2174

@diracdeltas @bbondy

### CODE REVIEW

File: app/extensions/brave/js/state/activeSettings.js [hunk 1]
```diff
@@ -2,98 +2,116 @@
...
+  if (blockAds && replaceAds && blockTracking) {
+   defaults.adControl = 'showBraveAds'
+  } else if (blockAds && !replaceAds && blockTracking) {
+   defaults.adControl = 'blockAds'
+  }
+  defaults.cookieControl = blockCookies ? 'block3rdPartyCookie' : 'allowAllCookies'
+
+  // TODO(bridiver) this should work just like the other bravery settings
+  let fingerprintingProtection = appState.get('settings').get('privacy.block-canvas-fingerprinting')
+  if (fingerprintingProtection !== 'boolean') {
```
Comment (bridiver) [inline app/extensions/brave/js/state/activeSettings.js]: fixing this...

File: app/extensions/brave/js/state/activeSettings.js [hunk 2]
```diff
@@ -2,98 +2,116 @@
...
-        return siteSettings.get('noScript')
+  settings.shieldsUp = (() => {
+    if (siteSettings) {
+      if (typeof siteSettings.get('shieldsUp') === 'boolean') {
+        return siteSettings.get('shieldsUp')
       }
     }
 
-    return appConfig.noScript.enabled
+    return typeof appSettings.get('shieldsUp') === 'boolean' ? appSettings.get('shieldsUp') : true
```
Comment (diracdeltas) [inline app/extensions/brave/js/state/activeSettings.js]: isn't `appSettings.get('shieldsUp')` always going to be `undefined`?

Comment (bridiver) [inline app/extensions/brave/js/state/activeSettings.js]: I guess so. I must have mixed this up with something else

**Comment by diracdeltas** [comment]
thanks!

**Pull request merged**